sklearn/utils/tests/test_extmath.py

```diff
@@ -330,6 +331,31 @@ def test_randomized_svd_sign_flip():
         assert_almost_equal(np.dot(v2.T, v2), np.eye(2))
 
 
+def test_randomized_svd_sign_flip_with_transpose():
+    # Check if flipping is always done based on u
```

Comment:
I would make this comment more detailed:

Check that randomized_svd flipping is always based on u irrespectively of transpose. See https://github.com/scikit-learn/scikit-learn/issues/5608 for more details.